utils.go

```diff
@@ -1,11 +1,8 @@
 package opslevel
 
 import (
-	"encoding/json"
-
-	"github.com/creasty/defaults"
 	"github.com/go-playground/validator/v10"
-	"gopkg.in/yaml.v3"
+	"github.com/taimoorgit/moredefaults"
```

Comment:
i would prefer we made this fork in the opslevel github org.  Also the long term goal being that we get the changes merged upstream.

Reply:
@rocktavious 100%, let's transfer it over before merging.

Reply:
Has been transferred: https://github.com/OpsLevel/moredefaults